src/OSPSuite.Presentation/Presenters/ContextMenus/ParameterIdentificationContextMenuItems.cs

```diff
@@ -85,10 +85,11 @@ public static IMenuBarItem RunParameterIdentification(ParameterIdentification pa
 
       public static IEnumerable<IMenuBarItem> ContextMenuItemsFor(ParameterIdentification parameterIdentification, IContainer container)
       {
-         yield return RunParameterIdentification(parameterIdentification, container);
+         yield return EditParameterIdentification(parameterIdentification, container);
+            
 
-         yield return EditParameterIdentification(parameterIdentification, container)
-            .AsGroupStarter();
+         yield return RunParameterIdentification(parameterIdentification, container)
```

Comment:
so double click just automatically executes the command behind the **first subnode**?
weird.

Reply:
Yeah, it's always been that way. Unless you override that behavior like we do for folders and modules where they expand the node.